chocolatey-diff/Public/Get-FullPackageInfo.ps1

```diff
@@ -19,24 +19,27 @@ function Get-FullPackageInfo {
         [string] $PackageId
     )
 
-    $pkgInfo = @{
-        Id       = $PackageId
-        Url      = "$(Get-ChocolateyPackageGalleryUrl)/${$PackageId}"
-        Versions = @()
-    }
+    if (!(Get-Variable -Name pkgInfo -Scope Script -ErrorAction SilentlyContinue)) {
```

Comment:
Tbh I'd just `if (-not $script:pkgInfo)` for this kind of thing. Cmdlet calls for this feel a little overkill. 🤔

Reply:
Changed